src/Template/PrivateMessages/folder.ctp

```diff
@@ -66,10 +66,11 @@ $this->set('title_for_layout', $this->Pages->formatTitle(
                 <h2 flex>
                     <?php
                     $n = $this->Paginator->param('count');
-                    echo $this->Number->format($__n('{folderName} ({n}&nbsp;message)',
+                    $this->Number->format($n)
+                    echo format(__n('{folderName} ({n}&nbsp;message)',
                                     '{folderName} ({n}&nbsp;messages)',
                                     $n, true),
-                                compact('folderName', 'n'));
+                                    ['folderName'=> $folderName, 'n'=> $this->Number->format($n)];
```

Comment:
You are missing a closing `)` between `]` and `;` at the end of the line.

Reply:
Thanks, Andi! I'm getting this now

> syntax error, unexpected ''{folderName} ({n}&nbsp;messag' (T_CONSTANT_ENCAPSED_STRING), expecting ',' or ')'

> Error in: ROOT\src\Template\PrivateMessages\folder.ctp, line 70


Line 70:

`$this->Number->format '{folderName} ({n}&nbsp;messages)',
                                    $n, true),
                                ['folderName'=> $folderName, 'n'=> $this->Number->format($n)]);`

Reply:
The whole block should look like
````php
<?php
    $n = $this->Paginator->param('count');
    echo format(__n('{folderName} ({n}&nbsp;message)',
                    '{folderName} ({n}&nbsp;messages)',
                    $n, true),
                ['folderName'=> $folderName, 'n'=> $this->Number->format($n)]
    );
?>
````

The line starting with `$n = ...` gets the number of messages.

The `echo format(...` outputs the string. It uses the CakePHP helper function `__n()` to get the correct translation. So the result of this function will be either `{folderName} ({n}&nbsp;message)` or `{folderName} ({n}&nbsp;messages)` depending on the number of messages, i.e. the value stored in `$n`.
`{folderName}` and `{n}` are placeholders which we want to replace with the correct values. So we use another helper function `format` to do this. `folderName` will be replaced with the value stored in `$folderName` and `$n` will be replaced with the localized value stored in `$n`.

Reply:
Oh, I got it now... How silly I am!

The final result should be localized, not the "processing result". Thanks a lot, @AndiPersti 

I committed the changes - https://github.com/Tatoeba/tatoeba2/pull/2381/commits/5b7d9c11c61a558beae9491023adc773658c9b81